app/controllers/albums_controller.rb

```diff
@@ -1,214 +1,89 @@
-      @kategorin.update(name: params[:album_category][:name],text: params[:album_category][:text],visible: params[:album_category][:visible])
-      if @kategorin.save
-        @kategorier = AlbumCategory.unscoped.order('name desc')
-        flash.now[:notice] = 'Kategorin '+@kategorin.name+' uppdaterades till Bildgalleriet'
-      end
-    end
-    
-    if (params[:commit] == 'Ta bort kategori')&&(params[:album_category])    
-      @category = AlbumCategory.find_by_id(params[:album_category][:id]).destroy
-      @kategorier = AlbumCategory.all      
-      flash.now[:notice] = 'Kategorin togs bort'
-    end
-    
-    unless @subcategory
-      @subcategory = Subcategory.new
-    end    
-    unless @subcategories       
-      @subcategories = Subcategory.order('text desc')
-    end
-    
-    if (params[:commit] == "Spara ny underkategori") && (params[:subcategory][:text] != nil) 
-      @subcategory.update(text: params[:subcategory][:text])
-      if @subcategory.save        
-        flash.now[:notice] = 'Underkategorin '+@subcategory.text+' skapades till Bildgalleriet'
-        @subcategory = Subcategory.new()
-        
-      end      
-    end
-        
-    if (params[:commit] == 'Spara underkategori') && (params[:subcategory][:id].nil? == false)
-      @subcategory = Subcategory.find(params[:subcategory][:id].nil? == false)      
-      @subcategory.update(text: params[:subcategory][:text])
-      if @subcategory.save
-        @subcategories = Subcategory.unscoped.order('text desc')
-        flash.now[:notice] = 'Underkategorin '+@subcategory.text+' uppdaterades till Bildgalleriet'
-      end
-    end
-    
-    if (params[:commit] == 'Ta bort underkategori')&&(params[:subcategory][:id].nil? == false)    
-      @category = Subcategory.find(params[:subcategory][:id]).destroy
-      @subcategories = Subcategory.unscoped.order('text desc')      
-      flash.now[:notice] = 'Kategorin togs bort'
-    end
-  end 
-  
-  def show    
+
+  # TODO add Settings somewhere else
+
+  def show
     if (@album.images)
       @images = @album.images.order('foto_file_name asc')
     else
-      @images = nil      
-    end    
+      @images = nil
+    end
   end
-  
+
   def new
-    @album = Album.new
   end
-  
+
   def create
-      @album = Album.new(album_params)      
-      @album.update(author: current_user.profile)    
-      respond_to do |format|
-        if @album.save                        
-          format.html { redirect_to @album, notice: 'Albumet skapades!' }
-          format.json { render :json => @album, :status => :created, :location => @album }
-        else
-          format.html { render action: "new" }
-          format.json { render json: @album.errors, status: :unprocessable_entity }
-        end
-      end
+    @album.author = current_user.profile
+    if @album.save
+      redirect_to @album, notice: 'Albumet skapades!'
+    else
+      render action: "new"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.